src/js/components/Drop/StyledDrop.js

```diff
@@ -34,7 +34,6 @@ const StyledDrop = styled.div`
   position: fixed;
   z-index: ${props => props.theme.global.drop.zIndex};
   outline: none;
-  overflow: auto; //since we set max-height
```

Comment:
we can safely remove this since StyledDrop extends from Box and Box handles overflow already.